.ci/install_cri_containerd.sh

```diff
@@ -31,12 +30,8 @@ cri_containerd_tarball_version=$(get_version "externals.cri-containerd.version")
 cri_containerd_repo=$(get_version "externals.cri-containerd.url")
 
 echo "Get cri_containerd version"
-go get "${containerd_repository}"
-cri_containerd_version=$(
-	cd "${GOPATH}/src/${containerd_repository}";
-	git checkout "v${cri_containerd_tarball_version}" >&2;
-	cat vendor.conf | grep "github.com/containerd/cri" | awk '{print $2}';
-)
+cri_containerd_version_url="https://raw.githubusercontent.com/containerd/containerd/v${cri_containerd_tarball_version}/vendor.conf"
```

Comment:
heh, well, I reckon @jodh-intel would ask you to move this const nearer the top of the file, like it used to be ;-)

Reply:
23333, updated. ;).

Reply:
Hi @grahamwhaley I suddenly found that I couldn't move this const nearer the top of the file, it includes another variable `cri_containerd_tarball_version`.
that's why majority of CI failed. 😭
```
11:13:56 /tmp/jenkins/workspace/kata-containers-tests-ubuntu-18-04-PR/go/src/github.com/kata-containers/tests/integration/containerd/shimv2/../../../.ci/install_cri_containerd.sh: line 15: cri_containerd_tarball_version: unbound variable
```